Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "LinearMixingModels"
 uuid = "b8ce4b42-e81b-4a39-a84a-67f74a9a16dd"
 authors = ["Invenia Technical Computing Corporation"]
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
Does a compat update normally merit patch bumps? Bumping for every compat helper MR seems excessive

Reply:
It does indeed. It's important to propagate these kinds of changes through the ecosytem as quickly as possible, which requires version bumps. Fortunately, patch bumps are very unobtrusive (you get them all the time without noticing because noone upper bounds patch versions), so there's really no hard in making a _lot_ of releases (I think ChainRules might have version `0.x.80` at some point, and it doesn't seem to cause any problems).

Reply:
It doesn't look like the CompatHelper does this automatically. Should I be making an issue there, or was this maybe done explicitly as it isn't the convention in all packages?

Reply:
Ah, yes. It didn't used to, but it does now, but it's opt-in behaviour. This might be a good opportunity to upgrade the CompatHelper action to the most recent version, and to incude the automatic version bump option (see CompatHelper docs).